### ISSUE TITLE
Logging into RAD Lab UI is giving client side Application error

### ISSUE THREAD

**Issue by guptamukul-google** [GoogleCloudPlatform/rad-lab#151]
**Describe the bug**

**To Reproduce**
1. Go to RAD Lab UI
2. Click on 'Sign in with Google' button
3. Once successfully logged in, you will see that UI lands on `/deployments` page
```
Application error: a client-side exception has occurred (see the browser console for more information).
```
**Expected behavior**
There should be no UI/console errors when someone logs in but it should show list of deployments.

**Screenshots**
<img width="1678" alt="Screenshot 2023-05-15 at 4 33 52 PM" src="https://github.com/GoogleCloudPlatform/rad-lab/assets/9626165/81fc2df3-67a2-42fd-937d-3a0ec8df7c9f">

**Current Workaround**
Remove `/deployments` from the browser URL so that it lands on base URL `/` and that works as expected.

**Comment by acolver** [comment]
I can consistently reproduce:
- Navigate to https://rad-lab-ui-d099.uc.r.appspot.com/signin/
- Sign out if you are signed in
- Open Chrome dev console
- Right-click on the refresh button
- Select "Empty cache and hard reload" (see image below)
- Sign in with Google

<img width="551" alt="image" src="https://github.com/GoogleCloudPlatform/rad-lab/assets/69013624/78d5a22c-f23f-4092-a783-0265282ca8b5">

**Comment by guptamukul-google** [comment]
This issue is resolved as part of #152
